### PR TITLE
Revert "Fix GTFS editor lock"

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
Loading new feed into an editor got broken!

Reverts ibi-group/datatools-ui#815

### CODE REVIEW

File: lib/editor/actions/editor.js [hunk 1]
```diff
@@ -167,56 +167,32 @@ export function lockEditorFeedSource (feedId: string) {
...
- */
-export function stopLockTimer () {
-  return function (dispatch: dispatchFn, getState: getStateFn) {
-    stopCurrentTimer(getState())
-  }
-}
-
-/**
- * HTTP call to the feed lock endpoint.
+ * HTTP call to .
```
Comment (miles-grant-ibigroup) [inline lib/editor/actions/editor.js:176]: Is this what this comment was before #815?

File: lib/editor/components/GtfsEditor.js [hunk 2]
```diff
@@ -153,54 +141,25 @@ export default class GtfsEditor extends Component<Props, State> {
...
-    // Listen to the page visibilityChange event so we can check for things like editor lock status
-    // or pause/resume lock timers right away.
-    window.addEventListener(getNormalizedEvent('visibilitychange'), this.onVisibilityChange)
+    const unloadEvent = window.attachEvent ? 'onbeforeunload' : 'beforeunload'
+    window.addEventListener(unloadEvent, this.componentCleanUp)
   }
 
   componentWillUnmount () {
     // Run component clean-up
+    // console.log('componnent is unmounting')
```
Comment (miles-grant-ibigroup) [inline lib/editor/components/GtfsEditor.js:158]: I'm guessing this was here before #815

**Review by miles-grant-ibigroup — APPROVED** [review]
Assuming this is a automatic revert PR, so happy to ignore some weird things I've seen.

While testing this I did notice that #815 also introduced another bug where when refreshing while editing a pattern the map goes gray. Also, refreshing from the table editor does not bring you back to the timetable editor.

File: lib/editor/actions/editor.js (hunk 1 above)
Comment (binh-dam-ibigroup) [reply lib/editor/actions/editor.js:176]: Yes it is a straight revert to what was there before.

**Review by philip-cline — APPROVED** [review]

**Pull request merged**